README.md

```diff
@@ -611,6 +611,18 @@ array of `Config.ThumbnailFilterMode` tags, optional
 
 Defines filter modes that should be hidden in the thumbnails browser
 
+##### onToolChanged
+function, optional
+
+Defines what happens if there is a change to the current tool
```

Comment:
I wouldn't use the word "Define", I'd say "notifies" or "is raised when" or whatever standard JS language is for when an event occurs.